projects/packages/forms/src/blocks/field-image-select/edit.tsx

```diff
@@ -55,9 +53,12 @@ export default function ImageSelectFieldEdit( props ) {
 	const { addOption } = useAddImageOption( optionsBlock?.clientId );
 
 	const blockProps = useBlockProps( {
-		className: clsx( 'jetpack-field jetpack-field-image-select', {
-			'is-selected': isSelected || isInnerBlockSelected,
-		} ),
+		className: clsx(
+			'jetpack-field jetpack-field-image-select is-non-animated-label is-non-outlined-block',
+			{
+				'is-selected': isSelected || isInnerBlockSelected,
```

Comment:
BTW I think you could easily replace the custom `is-selected` with Gutenberg native selectors. Not a blocker for this PR obviously.

Reply:
You made me think, most of the classes are only relevant in the frontend, so I just removed them, including "isSelected".